daemon/config/README.md

```diff
@@ -6,24 +6,45 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/daemon/config/RE
 
 # Daemon configuration
 
-<details markdown="1"><summary>The daemon configuration file is read from `/etc/netdata/netdata.conf`.</summary>
-Depending on your installation method, Netdata will have been installed either directly under `/`, or under `/opt/netdata`. The paths mentioned here and in the documentation in general assume that your installation is under `/`. If it is not, you will find the exact same paths under `/opt/netdata` as well. (i.e. `/etc/netdata` will be `/opt/netdata/etc/netdata`).</details>
+<details>
+<summary>The daemon configuration file is read from /etc/netdata/netdata.conf.</summary>
 
-This config file **is not needed by default**. Netdata works fine out of the box without it. But it does allow you to adapt the general behavior of Netdata, in great detail. You can find all these settings, with their default values, by accessing the URL `https://netdata.server.hostname:19999/netdata.conf`. For example check the configuration file of [netdata.firehol.org](http://netdata.firehol.org/netdata.conf). HTTP access to this file is limited by default to private IPs, via the [web server access lists](/web/server/README.md#access-lists).
+Depending on your installation method, Netdata will have been installed either directly under `/`, or
+under `/opt/netdata`. The paths mentioned here and in the documentation in general assume that your installation is
```

Comment:
On `FreeBSD` we install in `/usr/local/`, I do no know what is happening on Mac, but I suggest to also specify that the final installation has relationship with OS.

Reply:
I am not changing this text in this PR. The PR goal regarding documentation: update option sections.